apps/backend/src/users/user.entity.ts

```diff
@@ -13,4 +16,19 @@ export class User {
 
   @Column()
   email: string;
+
+  @Column()
+  profilePicture: string;
```

Comment:
:wrench: Can we allow `profilePicture` to be `null`? We probably don't need/want the pictures of applicants

Reply:
Yes! We can go back and change this to allow the `profilePicture` to be `null`.